gordo_components/builder/build_model.py

```diff
@@ -268,44 +268,43 @@ def _build(self) -> Tuple[Optional[sklearn.base.BaseEstimator], dict]:
                         }
                     )
                     scores.update({metric: val})
+
+                cv_duration_sec = time.time() - start
             else:
                 logger.debug("Unable to score model, has no attribute 'predict'.")
 
-            cv_duration_sec = time.time() - start
-
-            # If cross_val_only, return the cv_scores and empty model.
-            if self.evaluation_config["cv_mode"] == "cross_val_only":
```

Comment:
is this gone on purpose? Seems like it is no longer possible to only do CV without building the whole model?

Reply:
eh, ya.. that's true. It was a bit confusing getting the metadata built up in an intermediate fashion, but I overlooked the fact this would result in building the whole model. I fix, I fix.